plasmapy/formulary/radiation.py

```diff
@@ -0,0 +1,159 @@
+"""
+Defines the Radiation module as part of the Plasma Formulary
+"""
+
+__all__ = [
+    "thermal_bremsstrahlung",
+]
+
+import astropy.constants as const
+import astropy.units as u
+import numpy as np
+
+from scipy.special import exp1
+from typing import List, Tuple, Union
+
+from plasmapy.formulary.parameters import plasma_frequency
+from plasmapy.particles import Particle
+from plasmapy.utils.decorators import validate_quantities
+from plasmapy.utils.exceptions import PhysicsError
+
+
+@validate_quantities(
+    frequencies={"can_be_negative": False},
+    ne={"can_be_negative": False},
+    ni={"can_be_negative": False},
+    Te={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+)
+def thermal_bremsstrahlung(
+    frequencies: u.Hz,
+    ne: u.m ** -3,
+    Te: u.K,
+    ni: u.m ** -3 = None,
```

Comment:
```suggestion
    ne: u.m ** -3,
    Te: u.K,
    ni: u.m ** -3 = None,
```
```suggestion
    n_e: u.m ** -3,
    T_e: u.K,
    n_i: u.m ** -3 = None,
```
A minor change, since we've been using underscore to represent subscripts in other parts of the formulary.

Reply:
Sure - done